model/client4.go

```diff
@@ -3443,8 +3443,8 @@ func (c *Client4) UpdateChannelNotifyProps(channelId, userId string, props map[s
 }
 
 // AddChannelMember adds user to channel and return a channel member.
-func (c *Client4) AddChannelMember(channelId, userId string) (*ChannelMember, *Response, error) {
-	requestBody := map[string]string{"user_id": userId}
+func (c *Client4) AddChannelMember(channelId, userId, categoryId string) (*ChannelMember, *Response, error) {
```

Comment:
Same as above. Let's add a new method with the `categoryID` parameter. Something like `AddChannelMemberWithCategory`.

Reply:
Got it! Thank you @agnivade 🙂